### ISSUE TITLE
systemd support overrides the user's choice of default target

### ISSUE THREAD

**Issue by cerebrate** [microsoft/WSL#8844]
### Version

Microsoft Windows [Version 10.0.22000.856]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.62.1-20220921-1-microsoft-custom-WSL2+

### Distro Version

Debian bookworm/sid

### Repro Steps

1. Start WSL with systemd support enabled.

### Expected Behavior

Upon starting up, systemd should activate the units required by the currently configured default target, as displayed by `systemctl get-default`. On this system at this time, that is _multi-user.target_ .

### Actual Behavior

Systemd starts up and activates the units required by _graphical.target_ . Upon investigation, this appears to be a result of the command line argument passed to systemd, as seen here:

```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.3  0.1 168856 13104 ?        Ss   00:57   0:01 /lib/systemd/systemd --unit=graphical.target
```

Specifying a target in this way significantly limits the users' ability to customize systemd behavior.

**Comment by cerebrate** [comment]
Another thought - the reason I switched my default to multi-user.target in the first place was to avoid, with certain packages installed, graphical.target from trying to start up window managers, graphical login screens, and other such things unlikely to play well in the WSL scenario.

**Comment by benhillis** [comment]
@cerebrate - you're absolutely right, will fix this pronto.

**Comment by cerebrate** [comment]
Fixed in 68.2, so guess I can close this now. Shiny!
